CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
  
 ## Bug Fixes
 
+ - PR #566 Added missing <random> include
```

Comment:
If this is truly a hotfix there needs to be a new section with cuDF v0.4.1 and this listed as the bug fix for that section. Adding it here hides the change made to cause the bug fix.

Reply:
In addition, the date for v0.4.1 should be the date that the merge will take place; if this is a hotfix.

Reply:
It's already merged into `branch-0.5`. I made this new PR per the request of @mt-jones.